test/testsets.json

```diff
@@ -41,6 +41,9 @@
     { "name": "test_fs_open_read_sync_3.js", "skip": ["nuttx"], "reason": "not implemented for nuttx" },
     { "name": "test_gpio_input.js", "skip": ["all"], "reason": "needs hardware" },
     { "name": "test_gpio_output.js", "skip": ["all"], "reason": "need user input"},
+    { "name": "test_https_get.js", "timeout": 40 },
+    { "name": "test_https_request_response.js", "timeout": 40 },
+    { "name": "test_https_timeout.js", "timeout": 40 },
```

Comment:
@akhilkedia Currently Tizen is the only platform that support https. And we use `testsets.json` to identify which tests we can run and expected to pass for each platform. Without any description, we and other developers would think it can be run on all platform and supposed to pass.  Could you please specify "skip" and "reason"? I prefer to specify like following:

```
  { "name": "test_https_get.js", "timeout": 40 }, "skip": ["all"], "reason": "Only tizen support https, and on tizen we test manually" },
```